### PR TITLE
pyrecorder 0.2.1 :snowflake:

### PR DESCRIPTION
**Pull request by lorepirri**
pyrecorder 0.2.1 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-4213] 
- dev_url: https://github.com/anyoptimization/pyrecorder
- pypi: https://pypi.org/project/pyrecorder/
- inspector: https://inspector.pypi.io/project/pyrecorder/0.2.1/

### Explanation of changes:

- new feedstock

### Notes for reviewers

- `opencv-python` -> `opencv-python-headless`
- ⚠️  `opencv` or `py-opencv` or `opencv-python-headless` are not detected by `pip check` therefore I patch the requirements (even though this is weird we should check https://github.com/AnacondaRecipes/opencv-feedstock/blob/main/recipe/meta.yaml )

[PKG-4213]: https://anaconda.atlassian.net/browse/PKG-4213?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python
+    - imageio
+    - matplotlib-base >=3
+    - numpy >=1.15
+    - opencv >=4.0.0.21
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]:
```suggestion
    - opencv-python-headless >=4.0.0.21
```

File: abs.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+upload_channels:
```
Comment (JeanChristopheMorinPerso) [inline abs.yaml:1]:
```suggestion
channels:
  - sfe1ed40
upload_channels:
```
To get access to opencv-python-headless.

Comment (lorepirri) [reply abs.yaml:1]: `upload_channels` should work also for sourcing the deps, I'll try first without `channels`

Comment (lorepirri) [reply abs.yaml:1]: it works

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Review by Jrice1317 — APPROVED** [review]

**Pull request merged**